### PR TITLE
Feature/legend changes

### PR DESCRIPTION
**Pull request by hectoruch**
image

### CODE REVIEW

**Comment by hectoruch** [comment]
- New icons (alerts, show and hide)
- News designs.

image

**Comment by hectoruch** [comment]
- Add new styles when the user disabled a layer.
- Change shape-info styles and links (on nav and legends)
- Eye icon.
- Functions to change eye icon and the others icons.

image

**Comment by hectoruch** [comment]
Now countries and global data can close all their options.

newaccordion

**Comment by hectoruch** [comment]
- Info icon animation.
- The close option can use when the layer is disabled.
- Fix tooltip errors with click action.
- Changed hover styles on close and subscriptions icons.
- Fix text width and align.
- Fix cursors styles.

**Comment by hectoruch** [comment]
Before:
image

After:
image

File: app/assets/javascripts/map/views/LegendView.js [hunk 1]
```diff
@@ -286,6 +294,14 @@ define([
...
     },
 
+    updateLegendHidden: function() {
+
+    },
+
+    toggleSelectedHidden: function() {
+
+    },
+
```
Comment (pjosh) [inline app/assets/javascripts/map/views/LegendView.js]: empty methods?

**Pull request merged**